staging/src/k8s.io/apiserver/pkg/server/options/authentication.go

```diff
@@ -202,6 +202,9 @@ type DelegatingAuthenticationOptions struct {
 
 	// CustomRoundTripperFn allows for specifying a middleware function for custom HTTP behaviour for the authentication webhook client.
 	CustomRoundTripperFn transport.WrapperFunc
+
+	// DisableAnonymous gives user an option to disable Anonymous authentication.
+	DisableAnonymous bool
```

Comment:
In this case, I think this negative phrasing is correct because it means that the zero-value of `false` does the same thing the struct used to do.